src/azure-cli/azure/cli/command_modules/monitor/operations/activity_log_alerts.py

```diff
@@ -2,192 +2,464 @@
+                 "The format is FIELD=VALUE[ and FIELD=VALUE...]" + '''
+        The possible values for the field are 'resourceId', 'category', 'caller',
+        'level', 'operationName', 'resourceGroup', 'resourceProvider', 'status',
+        'subStatus', 'resourceType', or anything beginning with 'properties'.
+        '''
+        )
+        args_schema.condition.Element = AAZStrArg()
+
+        args_schema.action_group_id = AAZListArg(
+            options=["--action-group", "-a"],
+            help="Add an action group. Accepts space-separated action group identifiers. "
+                 "The identifier can be the action group's name or its resource ID.",
+        )
+        args_schema.action_group_id.Element = AAZStrArg()
+
+        args_schema.webhook_properties_list = AAZCustomListArg(
+            options=['--webhook-properties', '-w'],
+            help="Space-separated webhook properties in 'key[=value]' format. "
+                 "These properties are associated with the action groups added in this command." + '''
+        For any webhook receiver in these action group, this data is appended to the webhook
+        payload. To attach different webhook properties to different action groups, add the
+        action groups in separate update-action commands.
+        '''
+        )
+        args_schema.webhook_properties_list.Element = AAZStrArg()
+
+        return args_schema
+
+    def pre_operations(self):
+        args = self.ctx.args
+        args.location = "Global"
+        process_condition_parameter_for_alert(args)
+        webhook_properties = process_webhook_properties(args)
+        if not has_value(args.scope_ui):
+            from msrestazure.tools import resource_id
+            from azure.cli.core.commands.client_factory import get_subscription_id
+            # args.scopes = [resource_id(subscription=get_subscription_id(self.cli_ctx),
+            #                            resource_group=args.resource_group)]
+            # service check
+            args.scopes = [resource_id(subscription=get_subscription_id(self.cli_ctx))]
+        else:
+            args.scopes = args.scope_ui.to_serialized_data()
+        if _get_alert_settings_for_alert(self, args.resource_group, args.activity_log_alert_name,
+                                         throw_if_missing=False):
+            raise ValidationError(
+                'The activity log alert {} already exists in resource group {}.'.format(args.activity_log_alert_name,
+                                                                                        args.resource_group))
+        if not has_value(args.all_of):
+            args.all_of = [{
+                "field": "category",
+                "equals": "ServiceHealth",
+            }]
+        # Add action groups
+        action_group_rids = set()
+        if has_value(args.action_group_id):
+            action_group_rids = _normalize_names_instance(self.cli_ctx, args.action_group_id.to_serialized_data(),
```

Comment:
Why not use AAZResourceIdArgFormat? Like this: https://github.com/Azure/azure-cli/blob/759bb130cc6736c80aab945a0902d13a75ce7b8f/src/azure-cli/azure/cli/command_modules/network/operations/nat.py#L18-L26